### DISCUSSION TITLE
Should we release 0.70.15?

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#92] [Patches]
## Released. The 0.70.15 is the latest patch for 0.70 the release crew will prepare as, with the release of 0.73.0, version 0.70 exited the support window. 

Current Release: [0.70.14](https://github.com/facebook/react-native/releases/tag/v0.70.14)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by cipolleschi** [comment]
Merge: https://github.com/facebook/react-native/pull/41278 or you can't release as CI can't be green.

**Reply by lunaleaps** [reply]
This was closed, I'm guessing we don't need this anymore @cipolleschi?

**Reply by lunaleaps** [reply]
Another q, I see we have a pick request for https://github.com/facebook/react-native/commit/479b4bb1ba504706613f71530a4ddcf86ff1bcd9 for 0.72, wondering if that should go in this too. 

I do see our ios test failing like so but it doesn't seem related https://app.circleci.com/pipelines/github/facebook/react-native/38736/workflows/c7324285-467a-48c3-a9f6-fc4aaabc2b4c/jobs/1281830

**Comment by erinleigh90** [comment]
Can we backport [facebook/react-native@462d93d](https://github.com/facebook/react-native/commit/462d93d9c18b453f0c14c15cf44c8d1de280edb9) to 0.70.15? (Removes the unset-value@1.0.0 dependency which has been flagged as vulnerable)

**Reply by lunaleaps** [reply]
I see that 0.70 has already been bumped: 
https://github.com/facebook/react-native/blob/cb170efdd346776aa6941512e1c23eb11c3f0a3a/packages/react-native-codegen/package.json#L23C6-L23C10
https://github.com/facebook/react-native/blob/0.70-stable/repo-config/package.json#L41

**Comment by AllanZhengYP** [comment]
Can we backport the `react-devtools-core` dependency upgrade(https://github.com/facebook/react-native/commit/d3e8b646a70dcc59bb69b9978cd612a1f29ff7cc) to 0.70? It is causing depandabot alert

**Reply by lunaleaps** [reply]
What's the vulnerability that dependabot is sharing? What release fixes the vulnerability, why 4.27.7 specifically? 
https://github.com/facebook/react/blob/3e00e58a6ac7f73a3660f31d3129fb06d344167e/packages/react-devtools/CHANGELOG.md
or maybe better to just look at `react-devtools-core` https://github.com/facebook/react/commits/main/packages/react-devtools-core?before=3e00e58a6ac7f73a3660f31d3129fb06d344167e+35

**Comment by liamjones** [comment]
Oops, I forgot 0.70 was still in support too...

It was suggested that the webp vulnerability patched in Fresco should probably be backported to supported versions of RN - just adding a note here for reference: https://github.com/react-native-community/discussions-and-proposals/issues/725#issuecomment-1780222618

**Reply by lunaleaps** [reply]
It was decided to not backport this as its a major version upgrade and the advice is to upgrade to 0.73. Please keep the discussion on https://github.com/react-native-community/discussions-and-proposals/issues/725

**Comment by hurali97** [comment]
Need to merge this PR for CLI bump: https://github.com/facebook/react-native/pull/41479

**Reply by lunaleaps** [reply]
✅ picked

**Comment by cortinico** [comment]
headsup: with the release of 0.73.0, this minor (0.70) now moves into unsupported status, so don't expect further releases to be done. Check out [the release policy](https://github.com/reactwg/react-native-releases#releases-support-policy) for more details.
